src/utils/ddb/ddb.c

```diff
@@ -1397,6 +1484,18 @@ ddb_commands_help(struct ddb_ctx *ctx)
 	ddb_print(ctx, "    [path]\n");
 	ddb_print(ctx, "\tOptional, VOS tree path to query.\n");
 	ddb_print(ctx, "\n");
+
+	/* Command: dtx_aggr */
+	ddb_print(ctx, "dtx_aggr <path>\n");
+	ddb_print(ctx, "\tAggregate DTX entries until a given timestamp or duration.\n");
+	ddb_print(ctx, "    <path>\n");
+	ddb_print(ctx, "\tVOS tree path to aggregate.\n");
```

Comment:
So if `container` path is specified, it will aggregate the committed DTX entries for the given `container`; otherwise, the path will for `pool`, and all the committed DTX entries within the given pool (for all containers) will be aggregated, right? Then `VOS tree path for the pool or container to be aggregated.\n` maybe more clear.

Reply:
> Currently only container path is supported. Aggregating all the containers of the vos pool will be supported with providing no path.
> 
> * [x]  Fix help

Fixed with commit 81ed63414